cpp/include/cudf/strings/string_view.hpp

```diff
@@ -363,11 +356,25 @@ class string_view {
    */
   string_view& operator=(string_view&&) = default;
 
+  /**
+   * @brief No-position value.
+   *
+   * Used when specifying or returning an invalid or unknown character position value.
+   */
+  static inline cudf::size_type const npos{-1};
```

Comment:
I am happy with the decision to use `-1`. I wanted to raise the question because that webpage was worded so specifically about why that value was chosen, but I agree it is an odd choice and `-1` feels better.